cpu/nrf5x_common/periph/uart.c

```diff
@@ -266,14 +318,15 @@ void uart_write(uart_t uart, const uint8_t *data, size_t len)
     else {
         _write_buf(uart, data, len);
     }
+#endif
 }
 
 void uart_poweron(uart_t uart)
 {
     assert(uart < UART_NUMOF);
 
     if (isr_ctx[uart].rx_cb) {
```

Comment:
Should this also be
```suggestion
    if (isr_ctx[uart].rx_cb | IS_USED(MODULE_PERIPH_UART_NONBLOCKING)) {
```
?

Reply:
Nope, this is unrelated to the TX isr, which is the one impacted by the non-blocking functionality.